database/models/tables.py

```diff
@@ -36,6 +36,24 @@ def add_new_artist(self):
         insert = insert_statement("artist", [], [])
         self.execute_query(insert)
 
+    def get_single_artist_info(self, artist_name):
+        select = select_statement("artist")[:-1]
+        _where = where("artist", "name", "=", f'"{artist_name}"')
+        statement = f"{select} {_where};" 
+        print(f"DEBUG SQL STATEMENT: {statement}")
+        query = self.execute_query(statement).fetchone()
+        return self._select_all_artist_data(query)
+    
+    def get_albums_for_artist(self, artist_name):
```

Comment:
This is great! I mentioned in the e-mail there might be specific data I needed for the search but you've already got all of that. Thanks so much!